tests/unittests/test_pulsar.py

```diff
@@ -0,0 +1,161 @@
+        test_dict = {'key1' : 'val1', 'key_2' : 'val_2'}
+        var = hubblestack.extmods.modules.pulsar._dict_update(dest, upd, recursive_update=True, merge_lists=False)
+        assert var == test_dict
+
+    def test_dict_update_for_classic_dictUpdate(self):
+        dest = {'key1' : 'val1'}
+        upd = {'key_2' : 'val_2'}
+        test_dict = {'key1' : 'val1', 'key_2' : 'val_2'}
+        var = hubblestack.extmods.modules.pulsar._dict_update(dest, upd, recursive_update=False, merge_lists=False)
+        assert var == test_dict
+
+    def test_dict_update_for_dest_TypeError(self):
+        dest = 'TestValue1'
+        upd = {'key_1' : 'val_1', 'key_2' : 'val_2'}
+        try:
+            var = hubblestack.extmods.modules.pulsar._dict_update(dest, upd, recursive_update=True, merge_lists=False)
+        except TypeError:
+            pass
+
+    def test_dict_update_for_upd_TypeError(self):
+        dest = {'key_1' : 'val_1', 'key_2' : 'val_2'}
+        upd = 'TestValue2'
+        try:
+            var = hubblestack.extmods.modules.pulsar._dict_update(dest, upd, recursive_update=True, merge_lists=False)
+        except TypeError:
+            pass
+
+    def test_dict_update_for_upd_TypeError(self):
+        dest = {'key_1' : 'val_1', 'key_2' : 'val_2'}
+        upd = 'TestValue2'
+        try:
+            var = hubblestack.extmods.modules.pulsar._dict_update(dest, upd, recursive_update=True, merge_lists=False)
+        except TypeError:
+            pass
+
+    def test_dict_update_for_upd_AttributeError(self):
+        dest = {'key_1' : 'val_1', 'key_2' : 'val_2'}
+        upd = {}
+        try:
+            var = hubblestack.extmods.modules.pulsar._dict_update(dest, upd, recursive_update=False, merge_lists=False)
+        except AttributeError:
+            pass
+
+
+    def test_process(self):
+        configfile='tests/unittests/resources/hubblestack_pulsar_config.yaml'
+        verbose = False
+        def config_get(value, default):
+            return default
+        __salt__ = {}
+        __salt__['config.get'] = config_get
+        hubblestack.extmods.modules.pulsar.__salt__ = __salt__
+        hubblestack.extmods.modules.pulsar.__opts__ = {}
+        var = hubblestack.extmods.modules.pulsar.process(configfile,verbose)
+        hubblestack.extmods.modules.pulsar.__salt__ = {}
+        assert var != 0
+
+
+    def test_process_for_regex_exclusion_list(self):
+        configfile='tests/unittests/resources/hubblestack_pulsar_config.yaml'
+        verbose = False
+        def config_get(value, default):
+            return default
+        __salt__ = {}
+        __salt__['config.get'] = config_get
+        hubblestack.extmods.modules.pulsar.__salt__ = __salt__
+        hubblestack.extmods.modules.pulsar.__opts__ = {}
+        hubblestack.extmods.modules.pulsar.process(configfile,verbose)
+        hubblestack.extmods.modules.pulsar.__salt__ = {}
+        assert hubblestack.extmods.modules.pulsar.CONFIG != 0
```

Comment:
Another 0 comparison. This file has a bunch of them as well. In this particular case you probably just want to check if it's empty.